File: JavaScript/fibonacci.js
let a = 0n;
console.log(a.toString());
let b = 1n;
console.log(b.toString());
let c = a + b;

do {
    console.log(c.toString());
    a = b;
    b = c;
    c = a + b;
} while (c <= 18446744073709551615);

console.log("Overflowed!");

File: Go/fibonacci.go
package main
import "fmt"

func main() {
    var a uint64 = 0;
    fmt.Println(a)
    var b uint64 = 1;
    fmt.Println(b)
    var c uint64 = a + b;

    for true {
        fmt.Println(c)
        a = b
        b = c
        c = a + b
        if c > b {
            continue
        }
        fmt.Println("Overflowed!")
        break
    }
}
